docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -1,68 +1,72 @@
 ====================
-ClickHouse Connector
+ClickHouse connector
 ====================
 
-The ClickHouse connector allows querying tables in an external
-`Yandex ClickHouse <https://clickhouse.tech/>`_ instance. This can be used to join data between different
-systems like ClickHouse and Hive, or between two different ClickHouse instances.
+The ClickHouse connector allows querying tables in an external `Yandex
+ClickHouse <https://clickhouse.tech/>`_ instance. This can be used to join data
+between different systems like ClickHouse and Hive, or between two different
+ClickHouse instances.
```

Comment:
please separate text formatting (that doens't affect rendered output) from other changes.

Reply:
Forgot about fixing the wording. Done now